docs/hub/spaces-config-reference.md

```diff
@@ -78,6 +78,10 @@ Authorized relative redirect path of the connected OAuth app. `/login/callback`
 Whether the Space iframe can be embedded in other websites.
 Defaults to false, i.e. Spaces *can* be embedded.
 
+**`startup_duration_timeout`**: _string_  
+Set a custom startup duration timeout for your Space. This is the maximum time your Space is allowed to start before it times out and is flagged as unhealthy.
+Defaults to 30 minutes, but any valid duration (like `1h`, `30m`) is acceptable.
```

Comment:
30mins is quite long already, i hope we don't have too many Spaces that need more than that...